packages/libs/src/services/web3Manager/Web3Manager.ts

```diff
@@ -85,11 +85,10 @@ export class Web3Manager {
       web3Config.internalWeb3Config?.web3ProviderEndpoints
     ) {
       // either user has external web3 but it's not configured, or doesn't have web3
-      this.web3 = new Web3(
-        this.provider(
-          web3Config.internalWeb3Config.web3ProviderEndpoints[0] as string,
-          10000
-        )
+      const gatewayweb3endpoint = web3Config.internalWeb3Config.web3ProviderEndpoints[0]!
```

Comment:
Worth consolidating this with the other method?

Reply:
good call